docs-build/deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+mkdir ./tmpdocclone
+cd ./tmpdocclone
+git clone https://github.com/${GH_ORG}/docs.git
+
+cd docs
+git checkout -b test${TRAVIS_BUILD_NUMBER}
+
+cp ../../build/cli-commands.md ./docs/using-appsody/cli-commands.md
+
+git add docs/using-appsody/cli-commands.md
+
+git commit -m "Travis build: $TRAVIS_BUILD_NUMBER"
+
+git remote add origin-2 https://${GH_TOKEN}@github.com/${GH_ORG}/docs.git
```

Comment:
I wonder if you can avoid this second remote if you use the GH_TOKEN in the original git clone. Might be worth a quick try.

Reply:
I tried locally and this would work, but we should review with the team to be consistent in the approach.